independent-projects/tools/devtools-testing/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartGenerationTest.java

```diff
@@ -276,15 +276,15 @@ private void checkDockerfilesWithMaven(Path projectDir) {
         assertThat(projectDir.resolve("src/main/docker/Dockerfile.jvm")).exists()
                 .satisfies(checkContains("./mvnw package"))
                 .satisfies(checkContains("docker build -f src/main/docker/Dockerfile.jvm"))
-                .satisfies(checkContains("registry.access.redhat.com/ubi8/ubi-minimal:8.4"))
```

Comment:
@ia3andy, about the tests it was necessary to adjust some assertions and add some new ones.
I don't know what `-Dsnap` does.

Reply:
@netodevel => https://quarkus.io/guides/extension-codestart#snapshot-testing